### PR TITLE
Use webpack-dev-server and HMR for development

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: src/lib/MiradorViewer.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import { v4 as uuid } from 'uuid';
-import { App } from '../components/App';
+import HotApp from '../components/App';
```
Comment (mejackreed) [inline src/lib/MiradorViewer.js:5]: I guess exporting a HotApp for a production build has a minimal footprint so it shouldn't be an issue?

https://github.com/gaearon/react-hot-loader#what-about-production

**Comment by mejackreed** [comment]
Also can we remove `concurrently` from the devdependencies?

File: src/lib/MiradorViewer.js (hunk 1 above)
Comment (cbeer) [reply src/lib/MiradorViewer.js:5]: That's the claim 🤷‍♂️

**Comment by cbeer** [comment]
> One question, should we update the `build` command to provide `NODE_ENV=production`? It seems to be suggested here: https://github.com/gaearon/react-hot-loader#what-about-production

I guess it would have been good if I read that part (at least after I figured out what all these pieces were..). I ended up passing in the webpack mode to babel, but if there's some reason we want separate control??

**Comment by mejackreed** [comment]
Yeah I'm not sure exactly, but I am running into issues running both `npm run build` and `NODE_ENV=production npm run build` I would have assumed that would have worked. I think this will cause our package publication to fail.

**Comment by cbeer** [comment]
What about with `2b3156c`? Or, can you say more  about the issues?

**Comment by mejackreed** [comment]
Seems like all is working well now. Thanks for making those changes!

**Pull request merged**